client/my-sites/plans/current-plan/jetpack-checklist/constants.ts

```diff
@@ -65,7 +65,17 @@ export const JETPACK_SECURITY_CHECKLIST_TASKS: ChecklistTasks = {
 };
 
 export const JETPACK_PERFORMANCE_CHECKLIST_TASKS: ChecklistTasks = {
-	// No tasks for this checklist yet…
+	jetpack_lazy_images: {
+		title: translate( 'Lazy Load Images' ),
+		description: translate(
+			"Improve your site's speed by only loading images when visible on the screen."
+		),
+		getUrl: ( siteSlug, completed ) =>
+			completed ? `/media/${ siteSlug }` : `/settings/performance/${ siteSlug }`,
+		completedButtonText: translate( 'Upload images' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **137** times:
`translate( 'Upload Images' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
@eeeeevon13 / @jeffgolenski Thoughts on casing and translation re-use?

Reply:
Good questions to ask — there seems to be a mixture of different casing in the checklist. We could very well revisit this question after merging and leave it inconsistent at first, just not to send out new translations.